frontend/src/views/Contributors.vue

```diff
@@ -0,0 +1,56 @@
+<template lang="pug">
+  div
+    h1 Contributors
+    h2(v-if="this.errorMessage") {{ errorMessage }}
```

Comment:
This should eventually be replaced with a global error handler component

Reply:
Okay, noted.